src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1730,6 +1730,14 @@ private DetectProperties() {
             )
             .setGroups(DetectGroup.PATHS, DetectGroup.GLOBAL)
             .build();
+    
+    public static final BooleanProperty BLACKDUCK_OFFLINE_MODE_FORCE_BDIO =
+        BooleanProperty.newBuilder("blackduck.offline.mode.force.bdio", false)
+            .setInfo("Offline Mode", DetectPropertyFromVersion.VERSION_4_2_0)
```

Comment:
feels like this version is too early? We'd want 8.5 maybe?

Reply:
Corrected, thanks!